test/testpaint/TestPaint.cpp

```diff
@@ -27,8 +27,8 @@ namespace TestPaint
 {
     void ResetEnvironment()
     {
-        gPaintInteractionType = VIEWPORT_INTERACTION_ITEM_RIDE;
-        gPaintSession.InteractionType = VIEWPORT_INTERACTION_ITEM_RIDE;
+        gPaintInteractionType = static_cast<uint8_t>(ViewportInteractionItem::Ride);
+        gPaintSession.InteractionType = ViewportInteractionItem::Ride;
```

Comment:
These two seem unused 🤔

Reply:
I would leave for now.

Reply:
I'm not sure what those tests are about, it seems only Linux (i686, portable) is running testpaint. I'll leave it in but change the static cast to EnumValue as suggested.

Reply:
Its meant to check that there have been no changes to the painting code for all rides compared to the original game (hence why its only x86 compatible). It doesn't quite work correctly at the moment due to other changes but I'm trying to get it running again on all platforms and in the process this part will be removed so don't worry too much about making this good.